Octo-Bot/packet-generator-bot/bot/main.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+
+from os import system
+
+# Internal Imports
+from utils.RandNetParamsGen import *
+from ReplayGenerator import *
+
+
```

Comment:
Can you a little documentation for this function?

Reply:
Added!